net-mail/dot-forward/files/0.71-fix-build-for-clang16.patch

```diff
@@ -0,0 +1,1826 @@
+diff --git a/Makefile b/Makefile
+index 188d9e9..5d659c0 100644
+--- a/Makefile
++++ b/Makefile
+@@ -19,8 +19,8 @@ compile alloc_re.c alloc.h byte.h
+ auto-ccld.sh: \
+ conf-cc conf-ld warn-auto.sh
+ 	( cat warn-auto.sh; \
+-	echo CC=\'`head -1 conf-cc`\'; \
+-	echo LD=\'`head -1 conf-ld`\' \
++	echo CC=\'`head -n 1 conf-cc`\'; \
++	echo LD=\'`head -n 1 conf-ld`\' \
```

Comment:
nit: the head changes here are actually from the eclass thing we call in the ebuild. I'd reapply the patch with `git add -p` so we don't include this, or maybe split this part into a separate patch and drop the use in the ebuild. Up to you.

It's not terrible to have it mixed here, but it's not ideal, as it's not related.